.vscode/settings.json

```diff
@@ -1,6 +1,7 @@
 {
     "git.mergeEditor": false,
     "go.testEnvVars": {
+        "CLIENTS_RBAC_BASE_URL": "http://localhost:8021/api/rbac/v1",
```

Comment:
I added this to allow debug the related tests from vscode. An internal mock rbac is started by the specific test by using this information that listen in a different port than the configuration provided by the template for development, so no collision.